src/formatDate.js

```diff
@@ -8,7 +8,23 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const separator = fromFormat[fromFormat.length - 1];
+  const newData = date.split(separator);
+  const newFormat = Array.from(toFormat);
+  const newSeparator = newFormat.pop();
+  const obj = {};
```

Comment:
It's good practice to use meaningful variable names. The variable `obj` could be named more descriptively, for example, `dateParts` to better represent its purpose.